FD limit at: 65535
zookeeper error on zoo_create, path /rtb-test/serviceClass/monitor/monitor: node exists

--------------------------[Exception thrown]---------------------------
time:   2015-06-05T16:43:31Z
type:   ML::Exception
pid:    10215; tid: 10215
what:   Zookeeper error on zoo_create, path /rtb-test/serviceClass/monitor/monitor: node exists
stack:
00: 0x0x7f9f0a920ecb at __cxa_throw + 0xbb in /home/ubuntu/rtbkit/build/x86_64/bin/../bin/libexception_hook.so + 0xecb/home/ubuntu/rtbkit/build/x86_64/bin/../bin/libexception_hook.so(__cxa_throw+0xbb) [0x7f9f0a920ecb]
01: 0x0x7f9f08fade63 at Datacratic::ZookeeperConnection::checkRes(int, int&, char const*, char const*) + 0xe3 in /home/ubuntu/rtbkit/build/x86_64/bin/../bin/libservices.so + 0x139e63/home/ubuntu/rtbkit/build/x86_64/bin/../bin/libservices.so(_ZN10Datacratic19ZookeeperConnection8checkResEiRiPKcS3_+0xe3) [0x7f9f08fade63]
02: 0x0x7f9f08fada61 at Datacratic::ZookeeperConnection::createNode(std::string const&, std::string const&, bool, bool, bool, bool) + 0x441 in /home/ubuntu/rtbkit/build/x86_64/bin/../bin/libservices.so + 0x139a61/home/ubuntu/rtbkit/build/x86_64/bin/../bin/libservices.so(_ZN10Datacratic19ZookeeperConnection10createNodeERKSsS2_bbbb+0x441) [0x7f9f08fada61]
03: 0x0x7f9f08f5aa85 at Datacratic::ZookeeperConfigurationService::setUnique(std::string const&, Json::Value const&) + 0x285 in /home/ubuntu/rtbkit/build/x86_64/bin/../bin/libservices.so + 0xe6a85/home/ubuntu/rtbkit/build/x86_64/bin/../bin/libservices.so(_ZN10Datacratic29ZookeeperConfigurationService9setUniqueERKSsRKN4Json5ValueE+0x285) [0x7f9f08f5aa85]
04: 0x0x7f9f08f454d4 at Datacratic::ServiceBase::registerServiceProvider(std::string const&, std::vector<std::string, std::allocator<std::string> > const&) + 0x1c4 in /home/ubuntu/rtbkit/build/x86_64/bin/../bin/libservices.so + 0xd14d4/home/ubuntu/rtbkit/build/x86_64/bin/../bin/libservices.so(_ZN10Datacratic11ServiceBase23registerServiceProviderERKSsRKSt6vectorISsSaISsEE+0x1c4) [0x7f9f08f454d4]
05: 0x0x7f9f0a2a574f at RTBKIT::MonitorEndpoint::init(std::vector<std::string, std::allocator<std::string> > const&) + 0x14f in /home/ubuntu/rtbkit/build/x86_64/bin/../bin/libmonitor_service.so + 0x974f/home/ubuntu/rtbkit/build/x86_64/bin/../bin/libmonitor_service.so(_ZN6RTBKIT15MonitorEndpoint4initERKSt6vectorISsSaISsEE+0x14f) [0x7f9f0a2a574f]
06: 0x0x4090ff at main + 0x4af in build/x86_64/bin/monitor_service_runner + 0x90ffbuild/x86_64/bin/monitor_service_runner(main+0x4af) [0x4090ff]
07: 0x0x7f9f0947c76d at __libc_start_main + 0xed in /lib/x86_64-linux-gnu/libc.so.6 + 0x2176d/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xed) [0x7f9f0947c76d]
08: 0x0x409625 in build/x86_64/bin/monitor_service_runner + 0x9625build/x86_64/bin/monitor_service_runner() [0x409625]

terminate called after throwing an instance of 'ML::Exception'
  what():  Zookeeper error on zoo_create, path /rtb-test/serviceClass/monitor/monitor: node exists
